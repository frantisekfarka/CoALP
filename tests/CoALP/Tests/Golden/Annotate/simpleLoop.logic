p(X) :- p(s(X)).
