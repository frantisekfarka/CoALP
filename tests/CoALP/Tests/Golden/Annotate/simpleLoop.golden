p(V_2, transform-func-1(V_-3)) :- p(s(V_2), V_-3).
